v2/docs/gs/params.md

```diff
@@ -94,6 +105,12 @@ The parameter `durability.mclass` specifies the media class for storing journal
 files.  In general, best performance is achieved by storing the journal files
 on the fastest media class configured for a KVDB.
 
+If `durability.mclass` is set to `auto`, HSE currently selects (applies)
+`pmem`, or `staging`, or `capacity`, in that order, depending on which
+is the highest media class configured for the KVDB.
```

Comment:
Change "depending on which is the highest media class configured" to something like "depending which media classes are configured".   The sentence already clarifies the order, and saying "highest" in addition to that is confusing.

Reply:
Agreed.  I'll update this and the other auto-related paragraphs accordingly.